File: SEM_V/Obliczenia naukowe/Lista_2/zad2.jl
#=
#   @author Mateusz Laskowski
=#

################# DRAW PLOT OF THE FUNCTION #################

using Plots;

in = linspace(0, 40, 250);

function ln(x)
  return log(exp(1), x);
end

function f(x)
  return (exp(1)^x) * ln(1 + exp(-x));
end

arr = [f(x) for x = in];

plot(arr);

png("juliaplot");

File: SEM_V/Obliczenia naukowe/Lista_2/zad2.py
#
#   @author Mateusz Laskowski
#

################# DRAW PLOT OF THE FUNCTION #################

import matplotlib.pyplot as plt
import numpy as np
from numpy import exp, log as ln

t = np.arange(0, 40, 0.2)
def func(x):
    return (exp(1)**x) * ln(1 + exp(-x))
plt.plot(t, func(t), 'r')
plt.show()